source/github-helpers/get-default-branch.ts

```diff
@@ -23,12 +24,16 @@ const getDefaultBranch = cache.function(async function (repository?: pageDetect.
 
 	if (arguments.length === 0 || JSON.stringify(repository) === JSON.stringify(getRepo())) {
 		if (pageDetect.isRepoHome()) {
-			const currentBranch = select('#branch-select-menu [data-menu-button]');
+			const currentBranch = await elementReady('[data-hotkey="w"]');
 			if (currentBranch) { // If missing, it'll default to the API call
-				return currentBranch.textContent!;
+				return currentBranch.title;
```

Comment:
It feels like we changed this in the past in the opposite direction, I don't know why. I may be wrong. @yakov116 remember anything?

Reply:
I don't recall anything about this other then using the feed link (which broke)